pkg/controller/operandrequest/reconcile_operand.go

```diff
@@ -99,6 +99,10 @@ func (r *ReconcileOperandRequest) reconcileOperand(requestInstance *operatorv1al
 				merr.Add(fmt.Errorf("the ClusterServiceVersion for Subscription %s is Failed", opdConfig.Name))
 				requestInstance.SetMemberStatus(operand.Name, operatorv1alpha1.OperatorFailed, "")
 				continue
+			} else if csv.Status.Phase != olmv1alpha1.CSVPhaseSucceeded {
```

Comment:
you can remove the else and use `if` directly here.

Reply:
done